app/models/katello/host/content_facet.rb

```diff
@@ -18,6 +18,8 @@ class ContentFacet < Katello::Model
       validates :host, :presence => true, :allow_blank => false
       validates_with Validators::ContentViewEnvironmentValidator
 
+      attr_accessible :content_view, :content_view_id, :lifecycle_environment, :lifecycle_environment_id, :host
```

Comment:
@jlsherrill updated to add the tests. I had to add both content_view and content_view_id (same with lifecycle_environment) here for passing tests in test/models/host/content_facet_test.rb but I think its fine to set all of these as accessible?